frontend/src/routes/AdminPage/index.js

```diff
@@ -164,3 +186,95 @@ class AdminPage extends Component {
 }
 
 export default AdminPage;
+
+const MyInnerForm = props => {
+  const {
+    values,
+    touched,
+    errors,
+    dirty,
+    isSubmitting,
+    handleChange,
+    handleBlur,
+    handleSubmit,
+    handleReset
+  } = props;
+  return (
+    <Form>
+      <CSRFToken />
+
+      <Field name="replyText" placeholder="Edit the reply text" type="text" />
+      <Field
+        name="description"
+        placeholder="Edit the description of the committee"
+        type="text"
+      />
+
+      <SubmitButton
+        onClick={handleSubmit}
+        type="submit"
+        disabled={isSubmitting}
+      >
+        Submit
+      </SubmitButton>
+    </Form>
+  );
+};
+
+const EditCommitteeForm = withFormik({
+  mapPropsToValues() {
+    return {
+      replyText: "write reply text here",
+      description: "description here"
+    };
+  },
+  handleSubmit(
+    values,
+    {
+      props: { committee, apiRoot },
+      resetForm,
+      setSubmitting,
+      setFieldValue
+    }
+  ) {
+    const committeeNames = {
+      webkom: "Webkom",
+      arrkom: "Arrkom",
+      bedkom: "Bedkom",
+      pr: "PR",
+      readme: "readme",
+      labamba: "LaBamba",
+      fagkom: "Fagkom",
+      koskom: "Koskom"
+    };
+
+    const submission = {
+      name: committeeNames[committee],
+      description: values.description,
+      response_label: values.replyText
+    };
+
+    console.log(submission);
+
+    fetch(`${apiRoot}/api/committee/`, {
+      method: "POST",
```

Comment:
Get the error `{"name":["committee` with this name already exists."]}`, so I need help on how to customize the api to allow updates. The way I thought should work based on previous success didn't.